react-components/src/architecture/concrete/observations/types.ts

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import {
+  CDF_TO_VIEWER_TRANSFORMATION,
+  type CustomObjectIntersection,
+  type Overlay3D
+} from '@cognite/reveal';
+import { type ObservationProperties } from './models';
+import { type Vector3 } from 'three';
+import { type FdmNode } from '../../../utilities/FdmSDK';
+import { type DomainObjectIntersection } from '../../base/domainObjectsHelpers/DomainObjectIntersection';
+
+export enum ObservationStatus {
+  Default,
+  PendingDeletion,
+  PendingCreation
+}
+
+export type Observation = {
+  properties: ObservationProperties;
+  fdmMetadata?: FdmNode<ObservationProperties>;
+  status: ObservationStatus;
+};
+
+export function createEmptyObservationProperties(point: Vector3): ObservationProperties {
+  const cdfPosition = point.clone().applyMatrix4(CDF_TO_VIEWER_TRANSFORMATION.clone().invert());
+  return { positionX: cdfPosition.x, positionY: cdfPosition.y, positionZ: cdfPosition.z };
+}
+
+export const observationMarker = Symbol('observationSymbol');
+
+export type ObservationIntersection = Omit<DomainObjectIntersection, 'userData'> & {
```

Comment:
You don't need this I hope, when you have done the other things I have commented

Reply:
I want to keep this - it introduces much better type safety